spec/requests/registrations_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe 'Registration', type: :request do
+  describe 'Email registration' do
+    let(:signup_url) { '/auth' }
+    let(:signup_params) do
+      {
+        email: 'jack.ibarra@rootstrap.com',
```

Comment:
Consider adding [Faker](https://github.com/faker-ruby/faker) for generating this values on the fly 🙂